input/core/build.gradle.kts

```diff
@@ -87,18 +88,29 @@ kotlin {
 //            implementation(libs.krono.kotlinx)
 //            implementation(coil.compose)
 //            implementation(coil.network.ktor)
+            api(compose.components.resources)
         }
 
         commonTest.dependencies {
             implementation(libs.kommander.coroutines)
             implementation(compose.uiTest)
         }
 
+        androidMain.dependencies {
+            implementation("androidx.activity:activity-ktx:1.10.1")?.because("We need ComponentActivity for android file picker")
```

Comment:
Consder adding this dependecy in versions, instead of hard coding it here. It will make updating this part a little bit tricky